staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/interface.go

```diff
@@ -0,0 +1,98 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package fairqueuing
+
+import (
+	"time"
+)
+
+// QueueSetFactory knows how to make QueueSet objects.  The request
+// management filter makes a QueueSet for each priority level.  The
+// clock, and any other test-facilitating infrastructure, to use in
+// each QueueSet is known to the factory, the client does not need to
+// pass this stuff in each call to NewQueueSet.
+type QueueSetFactory interface {
+	NewQueueSet(name string, concurrencyLimit, numQueues, queueLengthLimit, handSize int, requestWaitLimit time.Duration) (QueueSet, error)
+}
+
+// QueueSet is the abstraction for the queuing and dispatching
+// functionality of one non-exempt priority level.  It covers the
+// functionality described in the "Assignment to a Queue", "Queuing",
+// and "Dispatching" sections of
+// https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/20190228-priority-and-fairness.md
+// .  Some day we may have connections between priority levels, but
+// today is not that day.
+type QueueSet interface {
+	// SetConfiguration updates the configuration
+	SetConfiguration(concurrencyLimit, desiredNumQueues, queueLengthLimit, handSize int, requestWaitLimit time.Duration) error
+
+	// Quiesce controls whether this system is quiescing.  Passing a
```

Comment:
Please define what quiesce means in this context.

Reply:
Adding clarifying statement:
```
// Quiesce controls whether this system is quiescing ie:
// whether this system is halting processing requests.
```

Reply:
@aaron-prindle : I do not see your clarifying statement in the file; did you commit and push?

I think a more detailed clarification is in order.  Something like:

// A quiescing QueueSet drains as normal but does not admit any new requests.

Reply:
I've added the suggested text. I duplicated the `interface.go` comments into `queueset.go` originally but only updated this in the other location (which is why they were not here).  I've synced the Wait and Quiesce comments now